openslides/core/static/js/core/base.js

```diff
@@ -530,33 +530,96 @@ angular.module('OpenSlidesApp.core', [
+                    on: {
+                        instanceReady: function() {
+                            // add a listener to ckeditor that parses the clipboard content and, after the regular filter,
+                            // additionally strips out all empty <p> paragraphs
+                            // TODO: check all kind of clipboard html content if "isEmpty" is a reliable property
+                            this.on('paste', function(evt) {
+                                if (evt.data.type == 'html') {
+                                    var fragment = CKEDITOR.htmlParser.fragment.fromHtml(evt.data.dataValue);
+                                    var writer = new CKEDITOR.htmlParser.basicWriter();
+                                    // html content will now be in a dom-like structure inside 'fragment'.
+                                    this.filter.applyTo(fragment);
+                                    if (fragment.children) {
+                                        var new_content_children = [];
+                                        for (var i = 0; i < fragment.children.length; i++) {
+                                            var empty = true;
+                                            if (fragment.children[i].children){
+                                                for (var j = 0; j < fragment.children[i].children.length; j++) {
+                                                    var child = fragment.children[i].children[j];
+                                                    if (child.name != 'p' && child.name != 'br') {
+                                                        empty = false;
+                                                    } else if (child.isEmpty !== true) {
+                                                        empty = false;
+                                                    }
+                                                }
+                                                if (empty === false) {
+                                                    new_content_children.push(fragment.children[i]);
+                                                }
+                                            } else {
+                                                if (fragment.children[i].name != 'p' && fragment.children[i].name != 'br' &&
+                                                    fragment.children[i].isEmpty !== true){
+                                                    new_content_children.push(fragment.children[i]);
+                                                }
+                                            }
+                                        }
+                                        fragment.children = new_content_children;
+                                    }
+                                    fragment.writeHtml(writer);
+                                    evt.data.dataValue = writer.getHtml();
```

Comment:
return the re-created fragment without the empty `<p>` and `<br>` into the editor import processing (same as at the begin of the function: by ckeditor)